google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -94,6 +103,7 @@ def __init__(self, *args: Any) -> None:
 class InstanceMetadata:
     ip_addrs: Dict[str, Any]
     context: ssl.SSLContext
+    cert_expiration: datetime.datetime
```

Comment:
nit: Maybe just `expiration`? It should be the expiration of the data as a whole (both token/cert)

Reply:
fixed